contrib/utilities/check_indentation.sh

```diff
@@ -1,4 +1,32 @@
 #!/bin/sh
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2014 - 2016 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# This is a script that is used by the continuous integration servers
```

Comment:
yes, to indent your contribution you need to run `contrib/utilities/indent`, which is documented below